_posts/2019-08-08-kubernetes-gated-deployments.md

```diff
@@ -0,0 +1,245 @@
+---
+layout: post
+title: "Kubernetes Gated Deployments"
+date: 2019-08-08 09:00:00 -0700
+cover: /assets/images/kubernetes-gated-deployments/cover.jpg
+excerpt: Kubernetes Gated Deployments is a Kubernetes controller that facilitates automatic regression testing and canary analysis on Kubernetes deployments, that can detect and roll back the causal change without developer intervention. It is designed to augment existing deployment processes by analyzing key functionality and performance metrics associated with the application.
+authors:
+  - name: Steven Fu
+    title:
+    url:
+    photo:
+  - name: Satish Ravi
+    title:
+    url:
+    photo:
+  - name: Jacob Brooks
+    title:
+    url:
+    photo:
+  - name: Silas Boyd-Wickizer
+    title: Sr. Director of Engineering
+    url: https://github.com/silasbw
+    photo: https://avatars.githubusercontent.com/silasbw
+---
+
+At GoDaddy, teams frequently perform continuous deployments of their services to
+Kubernetes. When engineers make changes to these services, those changes may
```

Comment:
Changes to a service would always change the behavior of that service, right? If your change had no effect on the behavior of the service, why would you make the change?

I think what you mean to say is that "Changes to a service may introduce unintended changes that weren't caught by unit and integration tests, resulting in a negative..."